shared/src/metabase/mbql/schema.cljc

```diff
@@ -1387,7 +1389,9 @@
 ;; this is the reference like [:template-tag "whatever"], not the [[TemplateTag]] schema for when it's declared in
```

Comment:
maybe add `or [:template-tag {:id "template-tag-id"}]`?

Reply:
Changing it from `"whatever"` to `<whatever>` to make it clear it's not just a string. I don't think we need to re-enumerate the exact syntax there, since it's not the point of that comment.